lib/components/Search/Search.tsx

```diff
@@ -31,9 +31,7 @@ export const Search: React.FC<SearchProps> = ({ url, callback }) => {
     }
   }, [isOpen]);
 
-  const performSearch = (e: React.FormEvent<HTMLFormElement>) => {
-    e.preventDefault();
```

Comment:
Why was this removed?

Reply:
On Rules, we pass a callback through the megamenu to obtain the search result and redirect the user to the search page. If the user cmd+k searches, it will "redirect" to the same page with a different query param but not refresh the page results. 

The search originally refreshed the page but I removed that as part of the callback change, this is simply reverting it.